### ISSUE TITLE
Fix enable_webservice_routes option

### ISSUE THREAD

**Issue by mgaonach** [ispyb/py-ispyb#219]
The `enable_webservice_routes` setting was implemented as a DB Option (set in `AdminVar` table). 

This is problematic because it means all py-ispyb instances connected to the same database will have the same value, while the purpose of this option was being able to have two instances with different values:

- one instance with the webservice routes **disabled** accessible from outside the facility;
- one instance with the webservice routes **enabled** accessible only from inside the facility.

This option should be a environment variable, with default value = `False` if not present.

**Comment by stufisher** [comment]
Yes i already spotted this, we just need to make the option nested and add an instance id

**Comment by stufisher** [comment]
I think you are right, these ones would be better moved to settings, and then enabled via environment variable.
